lighthouse-core/scripts/compare-runs.js

```diff
@@ -205,14 +213,24 @@ function aggregateResults(name) {
   for (const lhrPath of glob.sync(`${outputDir}/**/lhr-*.json`)) {
     const lhrJson = fs.readFileSync(lhrPath, 'utf-8');
     /** @type {LH.Result} */
-    const lhr = JSON.parse(lhrJson);
+    const lhr = lhrJson && JSON.parse(lhrJson);
+    if (!lhr || !lhr.audits) {
+      console.warn(`could not parse ${lhrPath}`);
```

Comment:
This isn't strictly true, if `!lhr` was false, but `!lhr.audits` was true, this just implies that there were no `audits` parsed. It still may have been parsed fine, might just not have audits. This error might not be specific enough to care the diff, but it is misleading.